client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -75,9 +76,9 @@ private Stringifiers() {
     /**
      * Parses string to the appropriate value.
```

Comment:
Please rephrase with a `to convert` verb. Also, please get rid of `appropriate` word, which is too vanilla, be more precise instead.